docker/driver-loader-legacy/docker-entrypoint.sh

```diff
@@ -24,8 +24,10 @@ print_usage() {
 	echo "  docker run -i -t --privileged -v /root/.falco:/root/.falco -v /proc:/host/proc:ro -v /boot:/host/boot:ro -v /lib/modules:/host/lib/modules:ro -v /usr:/host/usr:ro -v /etc:/host/etc:ro falcosecurity/falco-driver-loader-legacy:latest [driver] [options]"
 	echo ""
 	echo "Available drivers:"
-	echo "  kmod           kernel module (default)"
+	echo "  auto	       leverage automatic driver selection logic (default)"
```

Comment:
Like for rpm/deb, automatic selection is the default.
In this case, there is no need to explicitly call 
```
/usr/bin/falcoctl driver config --type modern_ebpf --type ebpf --type kmod
```
since that is already the default falcoctl config shipped within Falco (see https://github.com/falcosecurity/falco/pull/3154/files#diff-8cef78de718faa0811f718db1971271a3a7172a72b015e10bc59dc8077c5d4e2). 
For dep and rpm, we always call it to eventually reset the falcoctl configuration upon package upgrade.